eslint.config.mjs

```diff
@@ -0,0 +1,18 @@
+import adfinisEmberAddonConfig from "@adfinis/eslint-config/ember-addon";
+import ember from "eslint-plugin-ember";
+
+export default [
+  {
+    ignores: ["blueprints/**", "node_modules/**"],
+  },
+  {
+    plugins: { ember },
+    settings: {
+      "import/internal-regex": "^ember-emeis/",
+    },
+    rules: {
+      "ember/no-settled-after-test-helper": "warn",
+    },
+  },
+  ...adfinisEmberAddonConfig,
```

Comment:
This has to go at the top of the config, otherwise it could overwrite your custom definitions from above.